### ISSUE TITLE
About loss

### ISSUE THREAD

**Issue by wuqi-coder** [IDKiro/CBDNet-pytorch#24]
`asym_loss = torch.mean(if_asym * torch.abs(0.3 - F.relu(gt_noise - est_noise)) * torch.pow(est_noise - gt_noise, 2))`

文章中asym是避免噪声低估，"gt - est"这儿会不会有些问题呀，感觉est大于gt的时候，惩罚值反而大了？

**Comment by IDKiro** [comment]
对，这里写错了，relu这里应该是个signal函数，现在这个损失函数会导致噪声估计网络直接失效
README里面说的实现错误就是指这个，主要是一直很忙，而且懒得重新训练，所以一直没搞，后来就忘了
我现在先改一下吧，有空我再重新训练

**Comment by IDKiro** [comment]
已经修了，如果有问题可以重新开issue，我先关掉了

**Comment by wuqi-coder** [comment]
好的，非常感谢
